beancount2ledger/cli.py

```diff
@@ -68,11 +71,24 @@ def cli():
         action="version",
         version=f"%(prog)s {beancount2ledger.__version__}")
 
-    args = parser.parse_args()
-    config = get_config(args.config)
-    output = beancount2ledger.convert_file(
-        args.file.name, args.format, config=config)
-    print(output)
+    with contextlib.ExitStack() as stack:
+        args = parser.parse_args()
+        in_file = args.file
+
+        # beancount loader does not accept file objects, hence to support reading from
+        # stdin we write its content to a tempfile and call the loader on its path
+        if in_file == "-":
+            tmpfile = stack.enter_context(
+                NamedTemporaryFile(prefix="tmp.beancount2ledger.",
+                                   mode="w",
+                                   encoding=locale.getpreferredencoding()))
```

Comment:
Beancount input files must be encoded in UTF8. Thus I suspect this is not strictly correct (at least on systems where the environment encoding is not UTF8).

Reply:
Oh, thanks, I didn't know that. I'll submit a separate pull-request to fix the encoding to utf-8 (and I'll be sorry about that).